bin/.travis/trusty/setup_ezplatform.sh

```diff
@@ -91,7 +91,7 @@ if [[ -n "${DEPENDENCY_PACKAGE_NAME}" ]]; then
 fi
 
 echo '> Run composer install inside docker app container'
-docker-compose exec app sh -c 'composer install --no-suggest --no-progress --no-interaction --prefer-dist --optimize-autoloader'
+docker-compose exec app sh -c 'php -d memory_limit='3G' /usr/local/bin/composer install --no-suggest --no-progress --no-interaction --prefer-dist --optimize-autoloader'
```

Comment:
Isn't this change unneeded as we now use env variable for this?
https://github.com/ezsystems/ezplatform/commit/add8d279f1d7b3b2666bb72935ff23c795bf812c

Reply:
Sadly this env variable does not take effect here for some reason. We had the error with the lack of memory with the default composer memory limit value in the message until I added this option here.